spark/src/main/scala/io/delta/tables/DeltaMergeBuilder.scala

```diff
@@ -105,6 +105,7 @@ import org.apache.spark.sql.internal.SQLConf
  *     .merge(
```

Comment:
Shall we add a comment about `withSchemaEvolution()` like the `whenNotMatchedBySource` and `whenMatched`?